include/timespec_util.h

```diff
@@ -0,0 +1,18 @@
+#ifndef TIMESPEC_UTIL_H
+#define TIMESPEC_UTIL_H
+
+#include <time.h>
+#include <stdbool.h>
+#include <stdint.h>
+
+#define TIMESPEC_NS_IN_S 1000000000L
```

Comment:
I read this as X many nanoseconds in a second, so it makes sense to me. I could see it either way.

Reply:
Well, the fact that we are talking about this is proof enough that the original naming was ambigous. I like the suggested naming, and updated the name of the constant to `TIMESPEC_NSEC_PER_SEC`, with using the `TIMESPEC` prefix since this is part of `timespec_util.h`'s interface.

Let me know if you think this is okay, or if I should just rename it to `NSEC_PER_SEC`.

Reply:
Yup, WFM!